### ISSUE TITLE
Scaling health with modded armor

### ISSUE THREAD

**Issue by Needmoremods** [SilentChaos512/ScalingHealth#144]
### Versions

Minecraft Version: 1.10.2
Scaling Health Version: 1.11.x-1.0.4-46
Silent Lib Version: 1.10.2-2.1.4-61

Just a possible note, I have been trying to use scaling health whilst giving myself modded armor with the generic.maxhealth attribute and it doesn't work.

### Expected behavior

Health stacking on top of scaling health with modded armor

### Actual behaviour

Armor that doesn't seem to add health no stacking occurs.

**Comment by SilentChaos512** [comment]
What mod is the armor from? Also try enabling Debug Mode in Scaling Health's config. This will show all health modifiers applied to your player.

**Comment by Needmoremods** [comment]
> What mod is the armor from? Also try enabling Debug Mode in Scaling Health's config. This will show all health modifiers applied to your player.

Oh it was simply spawned armor from command blocks.

**Comment by SilentChaos512** [comment]
Tested this on modded and vanilla 1.12.2, everything seems to be working. Are there any particular setups that seems to fail? Two that I tried are:
`/give @p minecraft:diamond_boots 1 0 {AttributeModifiers:[{Slot:"feet", AttributeName:"generic.maxHealth", Name: "generic.maxHealth", Amount:0.05, Operation:1, UUIDLeast:2, UUIDMost:2}]}`
and
`/give @p minecraft:diamond_chestplate 1 0 {AttributeModifiers:[{Slot:"chest", AttributeName:"generic.maxHealth", Name: "generic.maxHealth", Amount:20, Operation:0, UUIDLeast:1, UUIDMost:1}]}`
Both as which work as intended.
